build_test.go

```diff
@@ -1016,6 +1050,39 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("validating mixins", func() {
+			when("stack image mixins disagree", func() {
+				it.Before(func() {
+					h.AssertNil(t, defaultBuilderImage.SetLabel("io.buildpacks.stack.mixins", `["mixinA"]`))
+					h.AssertNil(t, fakeDefaultRunImage.SetLabel("io.buildpacks.stack.mixins", `["mixinA"]`))
+				})
+				it("returns an error", func() {
+					err := subject.Build(context.TODO(), BuildOptions{
+						Image:   "some/app",
+						Builder: builderName,
+					})
+
+					h.AssertError(t, err, "validating stack mixins: buildpack 'buildpack.id@buildpack.version' requires missing mixin(s): build:mixinY, mixinX, run:mixinZ")
```

Comment:
This appears to be testing the buildpack mixins rather than having differing mixins between the builder & run images, as appears to be desired by the context message.

Reply:
Good catch! I must have been on autopilot. Will fix.

Reply:
Done.